roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -8,6 +8,28 @@
   with_items: "{{ lookup('file', 'common-templates-'+ version +'.yaml').split('\n---\n') | select('search', '(^|\n)[^#]') | list }}"
   register: ct_status
 
+# Get all templates (including templates deployed by a previous version)
+- name: Fetching all templates to inject ownerReference
+  set_fact:
+    templates: "{{ lookup('k8s', api_version=ct_status.results[0].result.apiVersion, kind='template', label_selector='template.kubevirt.io/type=base') }}"
```

Comment:
This will also claim ownership of all user created base templates! You need to be more selective and only claim templates created by the operator.

Reply:
Notice the label selector `template.kubevirt.io/type=base`, do user created templates have this label as well?

Reply:
I assume they don't, are you worried about templates cloned by the user?

Reply:
`template.kubevirt.io/type=base` says nothing about ownership, it marks the template as a base template (as opposed to a template in general or template created from existing VM). The UI only displays templates with type=base. Including user defined ones.

Reply:
Do you have any suggestions on how to query for operator created templates?

Reply:
Take the list you got from the query and filter it based on the expected old owner reference?

Reply:
I'll try that, thanks

Reply:
Done